### ISSUE TITLE
Unexpected keyword argument 'partition_name' in python-sdk

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#1353]
**Describe the bug**
```

self = <milvus.client.grpc_handler.GrpcHandler object at 0x7f9afffeb6a0>, table_name = 'test_partition_jnkEX5KG', partition_tag = '\n', timeout = 10

    def drop_partition(self, table_name, partition_tag, timeout=10):
        """
        Drop specific partition under designated table.

        :param table_name: target table name.
        :type  table_name: str

        :param partition_tag: tag name of specific partition
        :type  partition_tag: str

        :param timeout: time waiting for response.
        :type  timeout: int

        :return:
            Status: indicate if operation is successful

        """
        request = Prepare.partition_param(
            table_name=table_name,
            partition_name=None,
>           tag=partition_tag)
E       TypeError: partition_param() got an unexpected keyword argument 'partition_name'

/usr/local/lib/python3.7/site-packages/milvus/client/grpc_handler.py:806: TypeError

```
**Steps/Code to reproduce behavior**

**Expected behavior**

**Environment details**

**Screenshots**

**Additional context**

**Comment by BossZou** [comment]
I see. I have published a latest milvus SDK package v0.2.58 to solve this issue. Please check it.  @del-zhenwu

**Comment by del-zhenwu** [comment]
fixed
